### PR TITLE
Further optimize SVGs

### PR DESCRIPTION
**Pull request by alrra**
| File | Size before | Size after | Reduction |
|:---|---:|---:|---:|
| `1.svg` |    16219 B |    16172 B | 47 B (0.29%) |
| `2.svg` |     6062 B |     6015 B | 47 B (0.78%) |
| `banner.svg` |    37714 B |    30458 B | 7256 B (19.24%) |
| `calendar-icon.svg` |      473 B |      394 B | 79 B (16.70%) |
| `chromelogo.svg` |     2973 B |     2914 B | 59 B (1.98%) |
| `clock.svg` |      496 B |      232 B | 264 B (53.23%) |
| `coffee.svg` |      241 B |      208 B | 33 B (13.69%) |
| `default.svg` |      243 B |      220 B | 23 B (9.47%) |
| `email.svg` |      180 B |      176 B | 4 B (2.22%) |
| `food.svg` |      308 B |      295 B | 13 B (4.22%) |
| `fullgoogle.svg` |     1516 B |     1332 B | 184 B (12.14%) |
| `g.svg` |      607 B |      582 B | 25 B (4.12%) |
| `github.svg` |      470 B |      419 B | 51 B (10.85%) |
| `register-icon.svg` |      356 B |      320 B | 36 B (10.11%) |
| `twitter.svg` |      380 B |      378 B | 2 B (0.53%) |
| `youtube.svg` |      325 B |      304 B | 21 B (6.46%) |
| | 68563 B | 60419 B | 8144 B (11.88%) |

### CODE REVIEW

**Comment by jakearchibald** [comment]
@alrra what tool did you use for this, out of curiosity?

File: src/assets/speakers/default.svg [hunk 1]
```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 62 62"><path fill="#9aa0a6" d="M31 34a10.3 10.3 0 1 0-10.3-10.3A10.3 10.3 0 0 0 31 34zm0 5.2c-6.9 0-20.7 3.4-20.7 10.3v12.8h41.4V49.5c0-6.9-13.8-10.3-20.7-10.3z" data-name="Path 61"/></svg>
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 620 620"><path fill="#9aa0a6" d="M31 34a10.3 10.3 0 1 0-10.3-10.3A10.3 10.3 0 0 0 31 34zm0 5.2c-7 0-20.7 3.4-20.7 10.3v12.8h41.4V49.5c0-7-13.8-10.3-20.7-10.3z"/></svg>
```
Comment (jakearchibald) [inline src/assets/speakers/default.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 62 62"><path fill="#9aa0a6" d="M31 34a10.3 10.3 0 1 0-10.3-10.3A10.3 10.3 0 0 0 31 34zm0 5.2c-7 0-20.7 3.4-20.7 10.3v12.8h41.4V49.5c0-7-13.8-10.3-20.7-10.3z"/></svg>
```

**Comment by jakearchibald** [comment]
@alrra thanks!

**Comment by alrra** [comment]
> `speakers/default.svg` is broken, but I'll fix it up.

@jakearchibald Oops, sorry about that! It seems I copied the wrong one. Here is what I intended:

```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 620 620"><path fill="#9aa0a6" d="M310 340a103 103 0 1 0-103-103 103 103 0 0 0 103 103zm0 52c-69 0-207 34-207 103v128h414V495c0-69-138-103-207-103z"/></svg>
```

> what tool did you use for this, out of curiosity?

Unfortunately, no one tool, they were tweaked semi-manually.

**Comment by jakearchibald** [comment]
Oh wow, well thanks!

**Pull request merged**